### ISSUE TITLE
#105997978------------- Links must be distinguishable without relying on color

### ISSUE THREAD

**Issue by khyatit13** [khyatit13/Coding-practice#1862]
<strong>PageName:</strong> FeedViewDetailsPage  

<strong>Source:</strong>`<a target="_blank" href="https://us-iotdev.arcgis.com/devusa3functest/ksoqrenrugvqxizs/streams/arcgis/rest/services/feed_automationTest_httpPoller_JSON_2024_02_08_0403/StreamServer?" data-url="https://us-iotdev.arcgis.com/devusa3functest/ksoqrenrugvqxizs/streams/arcgis/rest/services/feed_automationTest_httpPoller_JSON_2024_02_08_0403/StreamServer?token=9RK8DY0qvaqYcPFrrqGyoYFDXDccEfRwGocAgWn1h2QHvqS27CYPWeoGfDDhiWbouvrpPtFqOCpMftXaKvLAbf1OV5jkMmHXEJBTesSLghpcmU2cF7VENNxgCkeAIINOFVrXVsmZ8ygjXyYSNicxLWWYrhQqwNEe3DW3GmmNIqHKh3JrQEU9gRkcJkFTBhEjsjmdfvog-3nPy_3zsTVm2kaA7LlB5VUMOfhQUMgVmO__SwjmywJsCYBjkLAkSiysiDEKkn0KBKnde9uUp9_RcQ..">`

<strong>Element Location: </strong>.js-service-link > a[target="_blank"]

<strong>Impact: </strong>serious

<strong>Failure Summary:</strong> Fix any of the following:
  The link has insufficient color contrast of 1.3:1 with the surrounding text. (Minimum contrast is 3:1, link text: #00619b, surrounding text: #4c4c4c)
  The link has no styling (such as underline) to distinguish it from the surrounding text

<strong>Tags:</strong> cat.color,wcag2a,wcag141

<strong>HelpUrl:<strong>https://dequeuniversity.com/rules/axe/4.6/link-in-text-block?application=axeAPI
<strong>Screenshot:<strong>`C:\Automation\Workspace\Velocity\Screenshot\FeedViewDetailsPage\105997978_Links must be distinguishable without relying on color.png`
